src/main/java/com/example/onlinebookstore/model/Book.java

```diff
@@ -1,4 +1,83 @@
 package com.example.onlinebookstore.model;
 
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+
+@Entity
```

Comment:
```suggestion
@Entity
@Getter
@Setter
```

Remove get and set methods